app/scenes/more_channels/more_channels.js

```diff
@@ -197,6 +205,11 @@ class MoreChannels extends PureComponent {
         });
     };
 
+    onCreateChannel = async () => {
+        this.props.actions.goBack();
```

Comment:
Actually found a bug in the NavigationModal while reviewing this line. It prevented scenes from being pushed onto the modal route stack which is why you have to call ```goBack``` and then push the scene. Is this the intended behavior or do we want to push the scene onto the modal route stack so that we have the ability to navigate back to the channel listing?